soundata/datasets/dcase_bioacoustic.py

```diff
@@ -0,0 +1,628 @@
+        checksum="abce1818ba10436971bad0b6a3464aa6",
+        # unpack_directories=["URBAN-SED_v2.0.0"],
+    ),
+    "validation-classes": download_utils.RemoteFileMetadata(
+        filename="DCASE2022_task5_Validation_set_classes.csv",
+        url="https://zenodo.org/record/6482837/files/DCASE2022_task5_Validation_set_classes.csv?download=1",
+        checksum="0c05ff0c9e1662ff8958c4c812abffdb",
+        # unpack_directories=["URBAN-SED_v2.0.0"],
+    ),
+    "eval": download_utils.RemoteFileMetadata(
+        filename="Evaluation_set_5shots.zip",
+        url="https://zenodo.org/record/6517414/files/Evaluation_set_5shots.zip?download=1",
+        checksum="5212c0e133874bba1ee25c81ced0de99",
+        # unpack_directories=["URBAN-SED_v2.0.0"],
+    ),
+}
+
+LICENSE_INFO = "Creative Commons Attribution 4.0 International"
+
+
+class Clip(core.Clip):
+    """DCASE bioacoustic Clip class
+
+    Args:
+        clip_id (str): id of the clip
+
+    Attributes:
+        audio (np.ndarray, float): path to the audio file
+        audio_path (str): path to the audio file
+        clip_id (str): clip id
+        events (soundata.annotations.Events): sound events with start time, end time, label and confidence
+        split (str): subset the clip belongs to (for experiments): train, validate, or test
+    """
+
+    def __init__(self, clip_id, data_home, dataset_name, index, metadata):
+        super().__init__(clip_id, data_home, dataset_name, index, metadata)
+
+        self.audio_path = self.get_path("audio")
+        self.csv_path = self.get_path("csv")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The clip's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @property
+    def split(self):
+        """The data splits (e.g. train)
+
+        Returns
+            * str - split
+
+        """
+        return self._clip_metadata.get("split")
+
+    @property
+    def subdataset(self):
+        """The (sub)dataset
+
+        Returns
+            * str -subdataset
+
+        """
+        return self._clip_metadata.get("subdataset")
+
+    @core.cached_property
+    def events_classes(self) -> Optional[list]:
+        """The audio events
+
+        Returns
+            * list - list of the annotated events
+
+        """
+        return load_events_classes(self.csv_path)
+
+    @core.cached_property
+    def events(self) -> Optional[annotations.Events]:
+        """The audio events
+
+        Returns
+            * annotations.Events - audio event object
+
+        """
+        return load_events(self.csv_path)
+
+    @core.cached_property
+    def POSevents(self) -> Optional[annotations.Events]:
+        """The audio events for POS (positive) class
+
+        Returns
+            * annotations.Events - audio event object
+
+        """
+        return load_POSevents(self.csv_path)
```

Comment:
Should these be added in the `jams` as well? Also in the `Track` docstring?

Reply:
I am not sure whether I can put two event objects in jam?

Reply:
this leads to an error because we are resticting events to be annotator type. should we allow events to be list just as mirdata? @magdalenafuentes 
E               TypeError: events should be of type annotations.Events or annotations.MultiAnnotator
soundata/jams_utils.py:99: TypeError